app/src/main/java/com/nsc/covidscore/FragmentAdapter.java

```diff
@@ -0,0 +1,33 @@
+package com.nsc.covidscore;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentStatePagerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FragmentAdapter extends FragmentStatePagerAdapter {
+    private final List<Fragment> mFragmentList = new ArrayList<>();
+    private final List<String> mFragmentTitleList = new ArrayList<>();
+
+    public FragmentAdapter(FragmentManager fm) {
+        super(fm);
+    }
+
+    public void addFragment(Fragment fragment, String title){
+        mFragmentList.add(fragment);
```

Comment:
The setupView Pager method uses the addFragment method to add the fragments to an adapter just after the adapter is instantiated. Since fragments are created with getItem via the fragment's specific index in the list, I don't think having an extra fragment in the list, even if it somehow happened, would be an issue.

Reply:
Resolving for now. We can create a bug issue for this if I misinterpreted you here, but it shouldn't need to hold up the merge